### DISCUSSION TITLE
[Question] how can I get access from iPad?

### DISCUSSION THREAD

**Discussion by radostnomne** [coder/code-server#1789] [Q&A] [answered]
Hi
I have installed code-server via brew (Macos Catalina) and can't connect from ipad in my local network. How can I get access to code-server from iPad?

**Comment by nhooyr** [comment, marked as answer]
Please file future questions on our subreddit at https://www.reddit.com/r/codeserver/

Also give https://github.com/cdr/code-server/blob/master/doc/guide.md a read.

You just need to expose code-server on `0.0.0.0` and then access it from your iPad with your MacBook's IP.
